### ISSUE TITLE
Error: Undefined rule "stylelint-disable-reason"

### ISSUE THREAD

**Issue by whizark** [stylelint/stylelint#975]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

New `stylelint-disable-reason` rule introduced in styelint 5.2.0 doesn't work and an error occurs.
(Probably, the rule isn't imported in https://github.com/stylelint/stylelint/blob/5.2.0/src/rules/index.js)

> Which rule, if any, is this issue related to?

`stylelint-disable-reason`

> What CSS is needed to reproduce this issue?

test.css:
```css
.foo {}
```

> What stylelint configuration is needed to reproduce this issue?

.stylelintrc.json:
```json
{
    "rules": {
        "stylelint-disable-reason": [
            "always-before",
            {
                "severity": "error"
            }
        ]
    }
}
```

> Which version of stylelint are you using?

`5.2.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI with `stylelint --config .stylelintrc.json test.css`

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No.

> What did you expect to happen?

The error doesn't occur and the rule works.

> What actually happened (e.g. what warnings or errors you are getting)?

Error occurs:
```shell
Error: Undefined rule "stylelint-disable-reason"
    at exports.default (/home/user/.nvm/versions/node/v5.7.0/lib/node_modules/stylelint/dist/utils/configurationError.js:8:13)
    at /home/user/.nvm/versions/node/v5.7.0/lib/node_modules/stylelint/dist/postcssPlugin.js:100:47
    at Array.forEach (native)
    at /home/user/.nvm/versions/node/v5.7.0/lib/node_modules/stylelint/dist/postcssPlugin.js:98:33
```

**Comment by evilebottnawi** [comment]
@whizark already PR https://github.com/stylelint/stylelint/pull/974

**Comment by whizark** [comment]
@evilebottnawi Thank you for the notice! I close this issues.
